README.md

```diff
@@ -70,7 +70,7 @@ The command removes all the Kubernetes components associated with the chart and
 
 ## Changelog
 
-Notable chart changes are listed in the [CHANGELOG](https://github.com/datawire/ambassador-chart/CHANGELOG.md)
+Notable chart changes are listed in the [CHANGELOG](https://github.com/DevotedHealth/ambassador-chart/blob/master/CHANGELOG.md)
```

Comment:
I think the right URL would be https://github.com/datawire/ambassador-chart/blob/master/CHANGELOG.md, right?
```suggestion
Notable chart changes are listed in the [CHANGELOG](https://github.com/datawire/ambassador-chart/blob/master/CHANGELOG.md)
```